src/store/metadata/oxigraph/store.py

```diff
@@ -45,12 +46,37 @@ def setup(self):
         )
         configuration = (f"{oxigraph_url}/query", f"{oxigraph_url}/update")
         self.db = Dataset(store=SPARQLUpdateStore(*configuration))
-
+    
     def get_datasets(self) -> Optional[Dict]:  # pragma: no cover
-        """
-        Gets all datasets
-        """
-        raise NotImplementedError
+            """
+            Gets all datasets
+            """
+            graph = self.db
+
+            result: Graph = construct_datasets(graph)
+
+        # Serialize the graph into jsonld
+            data = json.loads(result.serialize(format="json-ld"))
+
+            # Use a context file to shape our jsonld, removing long form references
+            data = jsonld.flatten(
+                data, {"@context": constants.CONTEXT, "@type": ["dcat:Catalog","hydra:Collection"]}
+            )
+        
+            data["@graph"][0]["@type"] = ["dcat:Catalog","hydra:Collection"]
```

Comment:
Add a comment here to go back and remove the `@type` renaming once the context weirdness is resolved.